Documentation/ApiOverview/LinkBrowser/Linkhandler/RecordLinkHandler.rst

```diff
@@ -7,10 +7,15 @@
 The RecordLinkHandler
 =====================
 
-The RecordLinkHandler enables editors to link to single records, for example a
-single news record.
+The :php:`RecordLinkHandler` enables editors to link to single records, for
+example the detail page of a news record.
 
-It is implemented in class :php:`\TYPO3\CMS\Backend\LinkHandler\RecordLinkHandler`
+You can find examples here:
+
+*   :ref:`Browse records of a table <TableRecordLinkBrowserTutorials>`
+*   :ref:`Link browser example in tutorial in the news manual <ext_news:linkhandler>`
```

Comment:
```suggestion
*   :ref:`Link browser example in tutorial in the news extension manual <ext_news:linkhandler>`
```

First thought: Why do I need for news (=Neuigkeiten) a manual ;-) Ah, it's the extension.